<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OmgExperimentProgressBase</class>
 <widget class="QDialog" name="OmgExperimentProgressBase">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>507</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Model Queue</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/resources.qrc">
    <normaloff>:/om_icon_small.png</normaloff>:/om_icon_small.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="4">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QTableWidget" name="tblQueue">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="showGrid">
       <bool>false</bool>
      </property>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>Progress</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Status</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Taxon</string>
       </property>
      </column>
      <item row="0" column="1">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../resources/status_queued.png</normaloff>../resources/status_queued.png</iconset>
       </property>
      </item>
     </widget>
     <widget class="QGroupBox" name="grpLogMessages">
      <property name="title">
       <string>Log Messages:</string>
      </property>
      <layout class="QGridLayout">
       <property name="margin">
        <number>9</number>
       </property>
       <property name="spacing">
        <number>6</number>
       </property>
       <item row="0" column="0">
        <widget class="QTextBrowser" name="teLog"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="lblExperimentProgress">
     <property name="text">
      <string>Processing model 1</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QToolButton" name="toolShowLog">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/show_up.png</normaloff>:/show_up.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QToolButton" name="toolHideLog">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/hide_down.png</normaloff>:/hide_down.png</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Legend:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>393</width>
       <height>22</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Queued:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../resources/resources.qrc">:/status_queued.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Running:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../resources/resources.qrc">:/status_running.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Completed:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../resources/resources.qrc">:/status_complete.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Aborted:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../resources/resources.qrc">:/status_aborted.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QProgressBar" name="progressTotal">
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="2">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbnOk">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>splitter</zorder>
  <zorder>progressTotal</zorder>
  <zorder>toolShowLog</zorder>
  <zorder>lblExperimentProgress</zorder>
  <zorder>toolHideLog</zorder>
  <zorder>label_7</zorder>
 </widget>
 <tabstops>
  <tabstop>tblQueue</tabstop>
  <tabstop>teLog</tabstop>
  <tabstop>toolShowLog</tabstop>
  <tabstop>toolHideLog</tabstop>
  <tabstop>pbnCancel</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pbnOk</sender>
   <signal>clicked()</signal>
   <receiver>OmgExperimentProgressBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>368</x>
     <y>495</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolShowLog</sender>
   <signal>clicked(bool)</signal>
   <receiver>grpLogMessages</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>418</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel">
     <x>418</x>
     <y>361</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolHideLog</sender>
   <signal>clicked(bool)</signal>
   <receiver>grpLogMessages</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>452</x>
     <y>376</y>
    </hint>
    <hint type="destinationlabel">
     <x>454</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
